### PR TITLE
(fix) Program enrollment && regimen history not loading

### PR DESCRIPTION
**Pull request by makombe**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-care-panel-app/src/program-enrollment/program-enrollment.component.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const ProgramEnrollment: React.FC<ProgramEnrollmentProps> = ({ enrollments = [],
           id: `${enrollment.enrollmentUuid}`,
           ...enrollment,
           ...firstEncounter,
-          changeReasons: enrollment?.firstEncounter?.changeReasons?.join(', '),
+          changeReasons: Array.isArray(firstEncounter?.changeReasons) ? firstEncounter.changeReasons.join(', ') : '',
```
Comment (ojwanganto) [inline packages/esm-care-panel-app/src/program-enrollment/program-enrollment.component.tsx:88]: what does the endpoint provide?

**Review by ojwanganto — DISMISSED** [review]
Thanks, @makombe

**Comment by ojwanganto** [comment]
@donaldkibet - please help with review

**Review by donaldkibet — DISMISSED** [review]
LGTM, Thanks @makombe could we add Unit test to prevent regression

**Comment by makombe** [comment]
> LGTM, Thanks @makombe could we add Unit test to prevent regression

Sure let me do that. Thanks

**Pull request merged**